docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,44 @@
 Version history
 ===============
 
+5.1.9 (2016 April 11)
```

Comment:
The date is going to need updating but otherwise this looks ok to me now. Given you already have a separate bullet point for Gatan, I'd be inclined to leave the item @sbesson commented on where it is.